public/catch-js-errors.js

```diff
@@ -118,6 +121,23 @@
 					savedWindowOnError();
 				}
 			};
+		} else if ( localStorageAssignment === 'analytics' ) {
```

Comment:
Too late :/ That key is already deployed #6360 and is in everybody's LocalStorage :/